generated_interfaces.go

```diff
@@ -1364,6 +1364,14 @@ type Page interface {
 // If request gets a 'redirect' response, the request is successfully finished with the 'requestfinished' event, and a new
 // request is  issued to a redirected url.
 type Request interface {
+	// An object with all the request HTTP headers associated with this request. The header names are lower-cased.
+	AllHeaders() (map[string]string, error)
+	// An array with all the request HTTP headers associated with this request. Unlike Request.allHeaders(), header
+	// names are NOT lower-cased. Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
+	HeadersArray() ([]map[string]string, error)
```

Comment:
This is not the type which we have upstream. Upstream we have []struct{name: string, value: string}

Reply:
Ok will be fixed once will add tests